src/api/projects/mutations/addTranslationKeyToProject/index.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import createTranslationKey, {
```

Comment:
this should also be renaemd to `addTranslationKeyToProject`.

I’m also thinking this should live in `@models/Project`.
That’s what we do for `addTranslationValueToTranslationKey`, it’s in `TranslationKey` and not in `TranslationValue`, no? at least that’s how it is in the GraphQL schema.